### PR TITLE
Font Awesome icon name fixes

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd - origin version of this PR

- course-about sidebar info icon "fa-info-circle"
- ~~ccx enrollment remove icon "fa-times-circle"~~
- ~~ccx grading policy warning "fa-warning"~~
- wiki search add file "fa-plus-circle"
- wiki history paragraph info "fa-info-circle"

### CODE REVIEW

File: lms/templates/ccx/enrollment.html [hunk 1]
```diff
@@ -65,7 +65,7 @@
             <tr>
               <td>${member.student}</td>
               <td>${member.student.email}</td>
-              <td><div class="revoke"><i class="icon-remove-sign"></i> Revoke access</div></td>
+              <td><div class="revoke"><i class="icon-times-circle"></i> Revoke access</div></td>
```
Comment (stvstnfrd) [inline lms/templates/ccx/enrollment.html]: I though FA4 used the `fa-*` convention?

Comment (caesar2164) [inline lms/templates/ccx/enrollment.html]: you are correct, sir

**Comment by caesar2164** [comment]
@stvstnfrd - removed ccx t/l from commit

**Comment by stvstnfrd** [comment]
@caesar2164 assuming this is good for us to pull in, minus the the CCX stuff, right?

**Comment by caesar2164** [comment]
This PR doesn't actually have the CCX changes @stvstnfrd, I forgot to edit description when I removed them...

**Pull request closed without merging**